src/layout/header/DesktopHeader.tsx

```diff
@@ -20,6 +21,7 @@ export default function DesktopHeader() {
   } else {
```

Comment:
Maybe I'm missing something but where's the if for this else?

Reply:
On line 17